backend/test/Test/DashboardSpec.hs

```diff
@@ -160,8 +157,7 @@ gettersSpec = do
           getExperiments (Just $ SqlId 1) (Just compoundId) (Just targetId)
         liftIO $ do
           length erExperiments `shouldBe` 1
-          -- TODO impossible to compute mean based on python results
---          erMeanIC50 `shouldBe` Just 33
+          erMeanIC50 `shouldBe` Just 51.025965961684655
```

Comment:
I think it's more :proper: to actually call analysis here, but that's ok for now, my 33 was not proper either.